internal/cmd/list-alerts.go

```diff
@@ -55,12 +55,12 @@ var listAlertsCmd = &cobra.Command{
 		if listAlertsCmdSettings.id == "" {
 			user, err = client.GetCurrentUser(pagerduty.GetCurrentUserOptions{})
 			if err != nil {
-				return err
+				return wrap.Error(err, "It seems like the authtoken is not set correctly or outdated. Please update the authtoken in the .pd.yml file. If you don't know how to create your authtoken, this might help:\n https://support.pagerduty.com/docs/generating-api-keys#generating-a-personal-rest-api-key\n")
```

Comment:
```suggestion
				return wrap.Error(err, "it seems like the authtoken is not set correctly or outdated. Please update the authtoken in the .pd.yml file. If you don't know how to create your authtoken, this might help:\n https://support.pagerduty.com/docs/generating-api-keys#generating-a-personal-rest-api-key\n")
```